app/signIn.tsx

```diff
@@ -1,9 +1,99 @@
-import { View } from "react-native";
+import { View, Text, TextInput, TouchableOpacity, StyleSheet, Alert } from 'react-native';
+import { Ionicons } from '@expo/vector-icons';
+import { useRouter } from 'expo-router';
+import { useState } from 'react';
 
 function SignInScreen() {
-  return(
-    <View>로그인</View>
+  const router = useRouter();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');  
+  const [passwordVisible, setPasswordVisible] = useState(false);
+
+  const handleLogin = async () => {
+    if (!email || !password) {
+      window.alert('이메일과 비밀번호를 모두 입력해주세요.');
+      return;
+    }
+
+    try {
+      const response = await fetch('https://thedayoftoday.kro.kr/signup', { 
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json', 'Accept': '*/*' },
+        body: JSON.stringify({ email, password }),
+      });
+
+      if (response.ok) {            
+        router.replace('/(tabs)');
+      } else {        
+        window.alert('이메일 또는 비밀번호를 확인해주세요.');
+      }
+    } catch (error) {           
+      window.alert('서버와의 연결에 실패했습니다.');
+    }
+  };
+
+  const handleSignUp = () => {
+    router.push('/signUp');
+  };
+
+  return (
+    <View style={styles.container}>
+      <Text style={styles.title}>오늘의 하루</Text>
+
+      <Text style={styles.loginLabel}>로그인</Text>
+
+      <TextInput
+        placeholder="이메일"
+        style={styles.input}
+        value={email}
+        onChangeText={setEmail}
+        keyboardType="email-address"
+        autoCapitalize="none"
+      />
+
+      <View style={styles.passwordContainer}>
+        <TextInput
+          placeholder="비밀번호"
+          style={styles.inputPassword}
+          value={password}
+          onChangeText={setPassword}
+          secureTextEntry={!passwordVisible}
+        />
+        
+        <TouchableOpacity onPress={() => setPasswordVisible(!passwordVisible)}>
+          <Ionicons 
+            name={passwordVisible ? 'eye-outline' : 'eye-off-outline'} 
+            size={20} 
+            color="#aaa" 
+          />
+        </TouchableOpacity>
+      </View>
+
+      <TouchableOpacity style={styles.loginButton} onPress={handleLogin}>
+        <Text style={styles.loginButtonText}>Login</Text>
+      </TouchableOpacity>
+
+      <View style={styles.signUpContainer}>
+        <Text>회원이 아니신가요? </Text>
+        <TouchableOpacity onPress={handleSignUp}>
+          <Text style={styles.signUpText}>회원가입하기</Text>
+        </TouchableOpacity>
+      </View>
+    </View>
   );
-};
+}
+
+const styles = StyleSheet.create({
+  container: { flex: 1, alignItems: 'center', justifyContent: 'center', padding: 20 },
```

Comment:
스타일 코드는 따로 분리해볼까요?? 또한 속성이 길어지는 경우와 가독성을 고려하여 모두 속성별로 줄바꿈하는 것이 좋을 것 같습니다.
```suggestion
  container: {
      flex: 1,
      alignItems: 'center',
      justifyContent: 'center',
      padding: 20
  },
```

Reply:
속성별로 줄바꿈해서 수정했습니다!